tests/unit/core/data/adapter/test_anomaly_adapter.py

```diff
@@ -31,13 +27,13 @@ def setup_method(self):
         self.val_data_roots: str = os.path.join(self.root_path, data_root_dict["val"])
         self.test_data_roots: str = os.path.join(self.root_path, data_root_dict["test"])
 
-        self.train_dataset_adapter = AnomalyClassificationDatasetAdapter(
+        self.train_dataset_adapter = AnomalyDatasetAdapter(
```

Comment:
Probably not a part of this PR but the only difference I see between these tests are the task types. Maybe we can just use pytest parameterize and reduce the classes? But this is optional.